src/lib/components/admin/Settings/Connections/OllamaConnection.svelte

```
@@ -4,11 +4,12 @@
 
 	import Tooltip from '$lib/components/common/Tooltip.svelte';
 	import SensitiveInput from '$lib/components/common/SensitiveInput.svelte';
-	import AddConnectionModal from './AddConnectionModal.svelte';
+	import AddConnectionModal from '$lib/components/AddConnectionModal.svelte';
 
 	import Cog6 from '$lib/components/icons/Cog6.svelte';
 	import Wrench from '$lib/components/icons/Wrench.svelte';
```

ESLint

[error] 'Wrench' is defined but never used.

(@typescript-eslint/no-unused-vars)